test/unit/context-manager/async-local-context-manager.test.js

```diff
@@ -5,15 +5,13 @@
 
 'use strict'
 
-const { test } = require('tap')
+const test = require('node:test')
 
 const runContextManagerTests = require('./context-manager-tests')
 const AsyncLocalContextManager = require('../../../lib/context-manager/async-local-context-manager')
 
-test('Async Local Context Manager', (t) => {
-  t.autoend()
-
-  runContextManagerTests(t, createContextManager)
```

Comment:
since we only have 1 context manager now you can move the context-manager-tests.js into here

Reply:
I had a similar thought, but erred on the side of least disruption here. Happy we agree that the extra file is no longer needed.